osgar/lib/pplanner.py

```diff
@@ -13,12 +15,17 @@ def find_path(img, start, finish, verbose=False):
     :param verbose: true for debugging
     :return: path (list of coordinates) or None if path not found
     """
-    max_x, max_y = img.shape
+    if len(img.shape) == 2:
+        # old 2D images - add extra dimension
+        img = np.expand_dims(img.copy(), axis=2)
+        start = (start[0], start[1], 0)
+        finish = [(x, y, 0) for x, y in finish]
+    max_x, max_y, max_z = img.shape
 
-    if not (0 <= start[0] < max_x and 0 <=start[1] < max_y):
+    if not (0 <= start[0] < max_x and 0 <= start[1] < max_y and 0 <= start[2] < max_z):
         return None
```

Comment:
yes it can happen now quite often. As you pointed out that 1024 is maybe not "good for everyone" - it corresponds to 500m so once you leave that area it will fail (i.e. you will not find corresponding voxel in the map).